doctr/__main__.py

```diff
@@ -89,8 +89,7 @@ def get_parser():
         public repositories for the user. This option is not recommended
         unless you are using a separate GitHub user for deploying.""")
     configure_parser.add_argument("--no-upload-key", action="store_false", default=True,
-        dest="upload_key", help="""Don't automatically upload the deploy key to GitHub. If you select this
-        option, you will not be prompted for your GitHub credentials. """)
+        dest="upload_key", help="""Don't automatically upload the deploy key to GitHub.""")
```

Comment:
All that's changed is that you'll always be prompted for a password, because `check_exists_repo` needs to be authenticated for private repos to work. The use-case you linked doesn't to be broken by the user having to login.